src/schemas/conseillers.schemas.ts

```diff
@@ -106,10 +106,112 @@ const validCandidatsStructure = Joi.object({
   diplome: Joi.string().error(new Error('Le filtre diplôme est invalide')),
 });
 
+const validCandidatureConseiller = Joi.object({
+  prenom: Joi.string().required().error(new Error('Le prénom est requis')),
+  nom: Joi.string().required().error(new Error('Le nom est requis')),
+  email: Joi.string()
+    .email()
+    .required()
+    .error(new Error('L’adresse e-mail est invalide')),
+  telephone: Joi.string()
+    .optional()
+    .allow('', null)
+    .pattern(/^(?:(?:\+)(33|590|596|594|262|269|687))(?:[\s.-]*\d{3}){3,4}$/)
+    .error(new Error('Le numéro de téléphone est invalide')),
+  nomCommune: Joi.string().required().error(new Error('La ville est requise')),
+  codePostal: Joi.string()
+    .required()
+    .min(5)
+    .max(5)
+    .error(new Error('Le code postal est invalide')),
+  codeCommune: Joi.string()
+    .required()
+    .min(4)
+    .max(5)
+    .error(new Error('Le code commune est invalide')),
+  location: Joi.object({
+    coordinates: Joi.array()
+      .required()
+      .items(Joi.number(), Joi.number())
+      .error(new Error('Les coordonées sont invalide')),
+    type: Joi.string().required().error(new Error('Le type est invalide')),
+  })
+    .required()
+    .messages({
+      'object.base': 'La location doit etre de type object',
```

Comment:
Est-ce qu'on garde ce message d'erreur ? Il ne me semble pas très explicite pour un utilisateur final

Reply:
Alors je l'avais ajouter quand je tester le it.each avec que des strings vide ; mais comme le undefined suffit , on peut l'enlever 👍 
Par contre même si on l'enlève , Joi gère déjà un message d'erreur par default :
![image](https://github.com/user-attachments/assets/5a558c64-4688-4efa-8b72-63516207fb1c)